projects/plugins/super-cache/wp-cache-phase2.php

```diff
@@ -2446,15 +2442,15 @@ function wp_cache_get_ob( &$buffer ) {
 		}
 
 		global $wp_super_cache_late_init;
-		if ( false == isset( $wp_super_cache_late_init ) || ( isset( $wp_super_cache_late_init ) && $wp_super_cache_late_init == 0 ) ) {
+		if ( ! isset( $wp_super_cache_late_init ) || $wp_super_cache_late_init === 0 ) {
 			wp_cache_add_to_buffer( $buffer, 'Super Cache dynamic page detected but late init not set. See the readme.txt for further details.' );
 		}
 
 		if ( $fr ) { // wpcache caching
 			wp_cache_debug( 'Writing dynamic buffer to wpcache file.' );
 			wp_cache_add_to_buffer( $buffer, 'Dynamic WPCache Super Cache' );
 			fwrite( $fr, '<?php die(); ?>' . $buffer );
-		} elseif ( isset( $fr2 ) ) { // supercache active
+		} else { // supercache active
```

Comment:
While this is equivalent to the code that was here before, it might be better to do like 
```suggestion
		} elseif ( $fr2 ) { // supercache active
```
since it looks like there are code paths above that could result in it being false here.